gdk/commands/Command.py

```diff
@@ -79,8 +78,10 @@ def _non_conflicting_args_map(self):
           _non_conflicting_args_map(dict): A dictionary object formed with argument as a key and a set of its non-conflicting
           args as value.
         """
+        from gdk.CLIParser import cli_tool
```

Comment:
The the variable was being accessed was not being recognized by the IDE I am using python 3.9.6 this fixes it.